server/index.js

```diff
@@ -50,7 +50,7 @@ export default class Server {
 
   handleRequest (req, res, parsedUrl) {
     // Parse url if parsedUrl not provided
-    if (!parsedUrl) {
+    if (!parsedUrl || typeof parsedUrl !== 'object') {
```

Comment:
This is a good point. But we throw an error with a proper message instead of parsing the url.

Because this is a mistake in the code.